docs/UserGuide.md

```diff
@@ -445,35 +441,35 @@ Examples:
 * `list` displays all students and lessons in TutorAid by only showing their names and list indexes.
 * `list -a` displays all students and lessons in TutorAid while showing all of their fields' data.
 
-### Adding students to lessons: `add -sl`
+### Adding student to lesson: `add -sl`
 {:.no_toc}
-Adds students to lessons.
+Adds a student to a lesson.
```

Comment:
Then maybe here can be "Tells TutorAid that a student attends a particular lesson"

Reply:
Updated it, thanks!